clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java

```diff
@@ -1251,4 +1317,8 @@ public String toString() {
     public Heartbeat heartbeat() {
         return heartbeat;
     }
+
+    public void setLastRebalanceTime(final long timestamp) {
```

Comment:
nit: seems this is for testing only? Should it be package private? Maybe also add comment `// for testing` ?

Reply:
The previous function is testing only too and we have a comment in line 1316.